src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/ControlDesigner.cs

```diff
@@ -85,7 +83,7 @@ private Point Location
         {
             Point loc = Control.Location;
 
-            ScrollableControl p = Control.Parent as ScrollableControl;
+            ScrollableControl? p = Control.Parent as ScrollableControl;
```

Comment:
Could we update this to have more descriptive name
```suggestion
            ScrollableControl? parent = Control.Parent as ScrollableControl;
```